src/rules/selector-attribute-quotes/index.js

```diff
@@ -0,0 +1,70 @@
+import {
+  isStandardSyntaxRule,
+  isStandardSyntaxSelector,
+  parseSelector,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-attribute-quotes"
+
+export const messages = ruleMessages(ruleName, {
+  expectedQuotes: "Expected quotes",
+  rejectedQuotes: "Unexpected quotes",
+})
+
+function isQuotesExist(str) {
```

Comment:
@jeddy3 It's not work with code:
```css
[class |= "top"] { }
```
Because we have space after operator. `postcss-selector-parser` return 
```shell
  ...
  raws: { unquoted: ' "top"' },
  quoted: false,
  ...
```